_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Hello^0^I am Yini He!
```

Comment:
I don't quite follow what "^0^" means? If you want it in can you put spaces around it?

Reply:
It is a smile icon which same with :) .